setup.cfg

```diff
@@ -57,7 +57,7 @@ gce =
 linode =
     linode-python; python_version<"3.0"
 lxc =
-    lxc-python2; python_version<"3.0"
+    lxc-python2; python_version<"3.0" and platform_system!="Darwin"
```

Comment:
I'd say linux only. LXC is *Linux* Containerization. It cannot work anywhere else